zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryResponseMapper.java

```diff
@@ -90,4 +93,53 @@ private static DecisionDefinitionItem toDecisionDefinition(final DecisionDefinit
         .decisionRequirementsName(d.decisionRequirementsName())
         .decisionRequirementsVersion(d.decisionRequirementsVersion());
   }
+
+  public static UserTaskSearchQueryResponse toUserTaskSearchQueryResponse(
+      final SearchQueryResult<UserTaskEntity> result) {
+    final var response = new UserTaskSearchQueryResponse();
+    final var total = result.total();
+    final var sortValues = result.sortValues();
+    final var items = result.items();
+
+    final var page = new SearchQueryPageResponse();
+    page.setTotalItems(total);
+    response.setPage(page);
+
+    if (sortValues != null) {
+      page.setLastSortValues(Arrays.asList(sortValues));
+    }
+
+    if (items != null) {
+      response.setItems(toUserTasks(items));
+    }
+
+    return response;
```

Comment:
🔧 Can we align this method's code with the other methods, e.g. `toProcessInstanceSearchQueryResponse`? This code can be made more concise by reusing existing methods, I think.